spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -19,59 +19,54 @@
 // John D. Ament
 // Emily Jiang
 
-[[release_notes_11]]
-== Release Notes for MicroProfile Config 1.1
+[[release_notes_20]]
+== Release Notes for MicroProfile Config 2.0
 
-The following changes occurred in the 1.1 release, compared to 1.0
+A full list of changes delivered in the 2.0 release can be found at link:https://github.com/eclipse/microprofile-config/milestone/8?closed=1[MicroProfile Config 2.0 Milestone].
 
-A full list of changes may be found on the link:https://github.com/eclipse/microprofile-config/milestone/2?closed=1[MicroProfile Config 1.1 Milestone]
+=== Incompatible Changes
+
+- ConfigSource#getPropertyNames is no longer a default method. The implementation of a ConfigSource must implement this method. (link:https://github.com/eclipse/microprofile-config/issues/431[431])
 
 === API/SPI Changes
 
-- The `ConfigSource` SPI has been extended with a default method that returns the property names for a given `ConfigSource`  (link:https://github.com/eclipse/microprofile-config/issues/178[#178])
+- Convenience methods have been added to Config allowing for the retrieval of multi-valued properties as lists instead of arrays (link:https://github.com/eclipse/microprofile-config/issues/496[#496])
+- Support Configuration Profile so that the corresponding properties associated with the active profile will be used (link:https://github.com/eclipse/microprofile-config/issues/418[#418])
+- Enable isolation of config properties into a separate POJO by introducing `@ConfigProperties` (link:https://github.com/eclipse/microprofile-config/issues/240[240])
+- Enable users to determine the winning source for a configuration value (link:https://github.com/eclipse/microprofile-config/issues/312[312]) (link:https://github.com/eclipse/microprofile-config/issues/43[43])
+- Expose conversion mechanism in Config API (link:https://github.com/eclipse/microprofile-config/issues/492[492])
 
 === Functional Changes
+- Provide built-in Converters: OptinalInt, OptionalLong and OptionalDouble (link:https://github.com/eclipse/microprofile-config/issues/513[513])
+- Clarify behaviour of Converters for primitive wrappers (link:https://github.com/eclipse/microprofile-config/issues/520[520])
```

Comment:
This wording requires the reader to chase and look for the effect of the clarification. I think it should state the outcome shortly, like 
```suggestion
- Clarifies that Converters for primitive wrappers apply to primitive types as well (link:https://github.com/eclipse/microprofile-config/issues/520[520])
```

Reply:
done